Cargo.lock

```diff
@@ -2712,14 +2595,12 @@ dependencies = [
 
 [[package]]
 name = "idna"
-version = "1.0.0"
+version = "0.5.0"
```

Comment:
Huh, weird. Probably not a big deal though...?

Reply:
Oh, not so nice after all:
https://github.com/servo/rust-url/pull/946
https://github.com/servo/rust-url/issues/937
:|